CONTRIBUTING.md

```diff
@@ -27,6 +47,67 @@ pull request reviewed, the reviewer(s) may ask you to complete additional
 work, tests, or other changes before your pull request can be ultimately
 accepted.
 
+# For maintainers
+
+## Setting up environment
+
+Prerequisites:
+
+- Dart SDK installed in the path, [standalone][dart_standalone_link], or included in the [Flutter SDK][dart_on_flutter_link].
+- [Mason CLI][mason_install_link] (to run and test the `bricks`);
+- [Node.js][node_js_dowload_link] (For the VS Code extension and the docs site);
+- Capability to run shell scripts (for the scripts under `tool/`).
+
+## Understanding the `packages/` contents:
```

Comment:
I think we could avoid this section here and moving any information to the package specific CONTRIBUTING, wdyt?

We're already referencing to them:
> "Some of the included projects have more specific instructions on contribution. In these cases, the project root may include a `CONTRIBUTING.md` file with such instructions. "

Reply:
I think the relationship between these packages is fundamental to understanding when/how to update them. The individual maintenance info is relevant too. Will add that paragraph.